main.cpp

```diff
@@ -56,14 +56,20 @@ Purpose:  This project will show you the difference between member functions and
 #include <string>
 struct T
 {
-    T(<#type name#> v, const char* <#variable name#>)   //1
-    //2
-    //3
+    T(char v, const char* vPointer);   //1
+    char value;   //2
+    std::string name; //3
 };
 
-struct <#structName1#>                                //4
+T::T(char v, const char* vPointer)
 {
-    <#type name#> compare(<#type name#> a, <#type name#> b) //5
+    value = v;
+    name = vPointer;
+}
```

Comment:
Use your constructor's member initializer list to init these member variables.
don't initialize them in the constructor body.
- [ ] revise